lib/VM/JSLib/Intl.cpp

```diff
@@ -434,7 +463,6 @@ CallResult<HermesValue> intlServiceConstructor(
     return ExecutionStatus::EXCEPTION;
   }
 
-  // NumberFormat has a couple extra checks to make.
```

Comment:
why is the comment gone?

Reply:
This function is a generic one for all the services .. not just NumberFormat .. But only NumberFormat had a service specific checks made earlier.. but this PR introduces DataFormat specific checks .. 
May be this comment can be made generic ..